bin/main

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env ruby
+
+puts 'Welcome to Ruby  Tic Tac Toe game '
+
+puts box = "+---+---+---+ \n| 1 | 2 | 3 | \n+---+---+---+ \n
+| 4 | 5 | 6 |\n+---+---+---+\n| 7 | 8 | 9 |\n+---+---+---+ \n"
+
+puts 'Lets start the game!'
+# players enter their name to begin the game
+puts "\nEnter Player 1 name: "
+one = gets.chomp.strip
+
+if one.empty?
+  puts "Player name cant be an empty,enter player's name"
+  one = gets.chomp
+elsif one.to_i.to_s.eql? one
+  puts 'Use only text no numbers: '
+  one = gets.chomp
+elsif one.length < 4
+  puts 'Player name should be more than 4 character: '
+  one = gets.chomp
+end
+
+puts 'Enter Player 1 name: '
+two = gets.chomp.strip
+
+if two.empty?
+  puts "Player name cant be an empty,enter player's name"
+  two = gets.chomp
+elsif two.to_i.to_s.eql? one
+  puts 'Use only text no numbers: '
+  two = gets.chomp
+elsif two.length < 4
+  puts 'Player name should be more than 4 character: '
+  two = gets.chomp
+elsif two == one
+  puts 'Players cannot have the same name'
+  two = gets.chomp
+end
+
+puts "#{one} will play with x and #{two} with 0"
+
+puts "\n #{one} starts the game"
+
+# the players begin to choose numbers from board
+board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+puts 'select an available cell from board'
+number = gets.chomp.to_i
+
+until board.include?(number)
+  puts 'invalid move. Please enter a number from 1-9'
+  number = gets.chomp.to_i
+end
+
+puts box
+puts "#{two} Its your turn"
+
+puts 'select an available cell from board'
+number = gets.chomp.to_i
+
+until board.include?(number)
+  puts 'invalid move. Please enter a number from 1-9'
+  number = gets.chomp.to_i
+end
+
+# determing the outcome of the game
+i = 9
+while i <= 9
+  puts box
+  puts "#{one} Its your turn"
+  puts count = gets.chomp.to_i
+  if [1, 2, 3].include? count
+    puts 'Game over!'
+    puts "The winner is #{one} !"
+  elsif [4, 5, 6].include? count
+    puts "The winner is #{two} !"
+  else
+    puts "It's tie! \n Game over"
+  end
+  i += 1
+end
```

Comment:
- [ ] use a terminating variable such as the one below

```
game_on = true

# initial instructions for player

while game_on
  # loop for each move

  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```

Reply:
Thank you for the feedback